.buildkite/pipeline.yml

```diff
@@ -256,8 +256,8 @@ steps:
       - helm repo add komodorio https://helm-charts.komodor.io
       - helm repo update
       - helm get values komodor-agent-staging -n komodor-staging > staging-values.yaml
+      - helm upgrade --install komodor-agent-staging komodorio/komodor-agent -f staging-values.yaml --set clusterName=komodor-staging-eks --namespace=komodor-staging --set imagePullSecret=docker-cfg-komodorio --set communications.serverHost=https://staging.app.komodor.com --set communications.tasksServerHost=https://staging.app.komodor.com --set communications.wsHost=wss://staging.app.komodor.com --set communications.mgmtServerHost=https://staging.app.komodor.com --set capabilities.telemetry.enable=false --set communications.telemetryServerHost=https://staging.telemetry.komodor.com --set apiKey=$$KOMODOR_AGENT_STAGING_API_KEY --set allowedResources.secret=true --set capabilities.event.redact="{.*KEY.*,.*key.*,.*BUGSNAG.*}"
```

Comment:
@Moran-k 
whats the point of passing values this way? if we already have a `staging-values` file?

Reply:
I was about to ask @Moran-k if we should delete the `staging-values.yaml` file in the CI folder.. See line 258 here, we generate it, and I assume it overwrites what we have in the folder, so I can only think it was some kind of test we committed by mistake

Reply:
Also I do think we should have files instead and do `-f current-values-from-get.yaml -f staging-values.yaml` to make it cleaner

Reply:
i see we generate it, but why do we do it ? we should just have values file per env

Reply:
the generated ones is our way of doing `--reuse-values` without the downside of reuse-values

Reply:
i know i know, but instead of running --reuse-values we can quite literally reuse the saved `values` file. anyways. lets ask @Moran-k  if there's any reason to do so and if not i would just delete it

Reply:
`staging-values.yaml` was not supposed to be committed,
As mick said the `get values` simulates the reuse same as we show to our customers now, so i did the same,
and i added all the parameters to the command to be safe if one day someone uninstalls the agent from the cluster or if we will recreate the cluster....

After should thinking, maybe it is better and clearer to remove the get values + all the `--set` and use a value file....

Reply:
i vote for using `-f staging-values` and removing `--set`s unless this is the only place where we are doing the get values update values trick (which is good to test somewhere)